SetVersion.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+target_dir=$1
+cd $target_dir
+mono External/Tools/SetVersion/SetVersion.exe FlashDevelop/Properties/AssemblyInfo.cs "%APPVEYOR_BUILD_NUMBER%"
```

Comment:
I'm not sure about the `%APPVEYOR_BUILD_NUMBER%` wouldn't it be "$APPVEYOR_BUILD_NUMBER" on linux / unix?